server/container/core/src/test/java/org/apache/james/core/MailImplTest.java

```diff
@@ -18,37 +18,34 @@
  ****************************************************************/
 package org.apache.james.core;
 
-import org.apache.mailet.base.test.MailUtil;
-import org.apache.mailet.MailAddress;
-import org.apache.mailet.Mail;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
 
 import javax.mail.MessagingException;
 import javax.mail.Session;
 import javax.mail.internet.MimeMessage;
 
-import java.io.ByteArrayInputStream;
-import java.util.ArrayList;
-import static org.junit.Assert.*;
+import org.apache.mailet.Mail;
+import org.apache.mailet.MailAddress;
+import org.apache.mailet.base.test.MailUtil;
 import org.junit.Test;
 
-public class MailImplTest extends MailTestAllImplementations {
+public class MailImplTest {
```

Comment:
If this is not a implementation of any abstract class/interface why the "Impl" in "MailImplTest" I would rename it to "MailTest"